### PR TITLE
[IMP] base: list sale_amazon in community with upgrade

### PR DESCRIPTION
**Pull request by niyasraphy**
before this commit, the sale_amazon module is not listed in community instance with upgrade button.

after this commit, the sale_amazon module will be listed in community apps list with upgrade button similar to sale_ebay module.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by niyasraphy** [comment]
hi @Demesmaeker ,
as we discussed here:  https://github.com/odoo/odoo/pull/109884#issuecomment-1396643478 , created the pr for adding the sale_amazon.

thanks

File: odoo/addons/base/data/ir_module_module.xml [hunk 1]
```diff
@@ -142,6 +142,20 @@
...
+            <field name="shortdesc">Amazon Connector</field>
+            <field name="sequence">320</field>
+            <field name="category_id" ref="base.module_category_sales_sales"/>
+            <field name="application" eval="True"/>
+            <field name="summary">Import Amazon orders and sync deliveries</field>
+            <field name="license">OEEL-1</field>
+            <field name="author">Odoo S.A.</field>
+            <field name="to_buy" eval="True"/>
+            <field name="icon">/base/static/img/icons/sale_amazon.png</field>
+            <field name="website">https://www.odoo.com/app/sales?utm_source=db&amp;utm_medium=module</field>
```
Comment (AntoineVDV) [inline odoo/addons/base/data/ir_module_module.xml:156]:
```suggestion
            <field name="website">https://www.odoo.com/app/amazon-connector?utm_source=db&amp;utm_medium=module</field>
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Hello @niyasraphy, thank you for this contribution :)

Please find the small comment below.

File: odoo/addons/base/data/ir_module_module.xml (hunk 1 above)
Comment (niyasraphy) [reply odoo/addons/base/data/ir_module_module.xml]: @AntoineVDV updated

**Review by AntoineVDV — APPROVED** [review]
Thanks :)

@robodoo r+

**Pull request closed without merging**